### DISCUSSION TITLE
May I know when will be planning to release next version ?

### DISCUSSION THREAD

**Discussion by manireddyk** [Netflix/dispatch#2749] [General]
Hi @kevgliss @mvilanova any planning to release next version this month/any expected date, 
We are planning to setup in production with new one. any release in this month do we have, so we will test and start using new version in our production

**Comment by mvilanova** [comment]
A new version will be released some time this week. This will be the last one for 2022.

**Reply by manireddyk** [reply]
Thank you @mvilanova , waiting for that :)

**Comment by SMillerDev** [comment]
Just over a year later, any word on the next version? It's been a while again.
